GLS edit of automated testing completed (#92)
rht-labs/tech-exercise: docs/slides/content/automated-testing.md

### Domain Driven Design
#### _How Does It Help?_
* Places the project's primary focus on the core domain and domain logic
* Bases complex designs on a model of the domain
* Initiates a creative collaboration between technical and domain experts to iteratively refine a conceptual model to address domain problems

> * Bridges the gap between domain experts and developers by using the same language to create the same understanding

Author: ```suggestion
* Bridges the gap between domain experts and developers by using the same language to create a shared understanding
```

> <!--TechEdit: I'd say "to create a shared understanding.">

Reviewer: feel free to update the sentence per your suggestion 😊

> <!--TechEdit: I'd say "to create a shared understanding.">

Author: ```suggestion

```

> <!--TechEdit: I'd say "to create a shared understanding.">

Author: ```suggestion

```

### Test Driven Development
#### _What Is It?_

> Test Driven Development (TDD) is a software development process that relies on the repetition of a very short development cycle.

Author: ```suggestion
Test Driven Development (TDD) is a software development process that relies on short development cycles.
```

> <!-- Tech Edit: I'm not sure what "the repetition of" means here. I'd suggest "...that relies on short development cycles" -->

Author: ```suggestion

```

> <!-- Tech Edit: I'm not sure what "the repetition of" means here. I'd suggest "...that relies on short development cycles" -->

Author: ```suggestion

```

### The Game Begins Again!

![TDD](images/bdd-tdd/lego-tdd-2.png) <!-- {.element: class="inline-image" style="height:500px"} -->
 - As a table group, pick a project to build. Projects must be something complex that has lots of different elements in it. Some examples, if you're stuck, might include a _zoo_, a _shopping center_, a _spaceship_, or a _city_.
 - All tests at the table must continue to pass at all times, regardless of which pair wrote them.  <!-- {.element: class="fragment" data-fragment-index="1"} -->
 - This time is that the implementation has to be integrated with the main project in the center of the table. <!-- {.element: class="fragment" data-fragment-index="2"} -->
"The goal is to integrate the elements that you built previously into the project selected by the table group." -->

> 

Reviewer: a new line is required here :)

### The Game Begins Again!

![TDD](images/bdd-tdd/lego-tdd-2.png) <!-- {.element: class="inline-image" style="height:500px"} -->
 - As a table group, pick a project to build. Projects must be something complex that has lots of different elements in it. Some examples, if you're stuck, might include a _zoo_, a _shopping center_, a _spaceship_, or a _city_.
 - All tests at the table must continue to pass at all times, regardless of which pair wrote them.  <!-- {.element: class="fragment" data-fragment-index="1"} -->
 - This time is that the implementation has to be integrated with the main project in the center of the table. <!-- {.element: class="fragment" data-fragment-index="2"} -->

> 

Reviewer: ```suggestion
```

> "The goal is to integrate the elements that you built previously into the project selected by the table group." -->

Reviewer: ```suggestion
- The goal is to integrate the elements that you built previously into the project selected by the table group.<!-- {.element: class="fragment" data-fragment-index="3"} -->
```

### The Game Begins Again!

![TDD](images/bdd-tdd/lego-tdd-2.png) <!-- {.element: class="inline-image" style="height:500px"} -->
 - As a table group, pick a project to build. Projects must be something complex that has lots of different elements in it. Some examples, if you're stuck, might include a _zoo_, a _shopping center_, a _spaceship_, or a _city_.
 - All tests at the table must continue to pass at all times, regardless of which pair wrote them.  <!-- {.element: class="fragment" data-fragment-index="1"} -->
 - This time is that the implementation has to be integrated with the main project in the center of the table. <!-- {.element: class="fragment" data-fragment-index="2"} -->

> <!--The final bullet above doesn't make sense. What implementation? The features built previously by the pairs? How about 

Author: ```suggestion

```

> "The goal is to integrate the features that you built previously in pairs into the project selected by the table group." -->

Reviewer: this sounds okay to me. Can we make it like this though?
```suggestion
"The goal is to integrate the elements that you built previously into the project selected by the table group." -->
```

> "The goal is to integrate the features that you built previously in pairs into the project selected by the table group." -->

Author: ```suggestion

```
